packages/docusaurus-theme-common/package.json

```diff
@@ -8,6 +8,8 @@
     "build": "node copyUntypedFiles.mjs && tsc",
     "watch": "node copyUntypedFiles.mjs && tsc --watch"
   },
+  "type": "module",
+  "sideEffects": false,
```

Comment:
does this thing produce failures alone?

It's something I wanted to test too, as it could help with tree shaking and code splitting

Reply:
Ah, `sideEffects` is good. I can send another PR to add it if it's worth it